Gemfile

```diff
@@ -8,11 +8,13 @@ gemspec
 gem "rake"
 
 group :development, :test do
+  gem "rails"
```

Comment:
do we need rails?

Reply:
- Yes we need, `railties` was not enough.